enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -1474,27 +1472,23 @@ func (c *FirecrackerContainer) getJailerConfig(ctx context.Context, kernelImageP
 		return nil, status.WrapError(err, "get cgroup version")
 	}
 
-	if c.cgroupSettings != nil {
-		if err := os.MkdirAll(c.cgroupPath(), 0755); err != nil {
-			return nil, status.UnavailableErrorf("create cgroup: %s", err)
-		}
-		if err := cgroup.Setup(ctx, c.cgroupPath(), c.cgroupSettings, c.blockDevice); err != nil {
-			return nil, status.UnavailableErrorf("setup cgroup: %s", err)
-		}
+	numaNode := 0
+	if c.cgroupSettings.NumaNode != nil {
+		numaNode = int(c.cgroupSettings.GetNumaNode())
 	}
-
 	return &fcclient.JailerConfig{
 		JailerBinary:   c.executorConfig.JailerBinaryPath,
 		ChrootBaseDir:  c.jailerRoot,
 		ID:             c.id,
 		UID:            fcclient.Int(unix.Geteuid()),
 		GID:            fcclient.Int(unix.Getegid()),
-		NumaNode:       fcclient.Int(c.numaNode),
+		NumaNode:       fcclient.Int(numaNode),
 		ExecFile:       c.executorConfig.FirecrackerBinaryPath,
 		ChrootStrategy: fcclient.NewNaiveChrootStrategy(kernelImagePath),
 		Stdout:         c.vmLogWriter(),
 		Stderr:         c.vmLogWriter(),
 		CgroupVersion:  cgroupVersion,
+		CgroupArgs:     []string{fmt.Sprintf("cpuset.cpus=%s", cpuset.Format(c.cgroupSettings.GetCpusetCpus()))},
```

Comment:
```suggestion
		// We normally set cpuset.cpus in cgroup.Setup(),
		// but the go SDK clobbers our setting when applying
		// the NUMA node setting. Override this manually
		// for now.
		CgroupArgs:     []string{fmt.Sprintf("cpuset.cpus=%s", cpuset.Format(c.cgroupSettings.GetCpusetCpus()))},
```

Reply:
Also I think we may need to set `cpuset.mems` to override what the go SDK is trying to set?

Reply:
Done, good call.